### PR TITLE
Feat/create-row-between-component

### PR DESCRIPTION
**Pull request by Leeosteves**
# 🚨 Fill in the fields below

[Create Row Between component](https://trello.com/c/D7z1NfRr/2-create-row-between-component)

## Description

```
~> Added Row between component to help us with the layout of the application
```

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Check list

- [x] PRODUCT

  - [x] Put PR link on ticket
  - [x] Move the ticket

- [x] DEVELOPERS

  - [x] My code follows the style guidelines of this project
 - [x] I have commented my code, particularly in hard-to-understand areas
  - [x] I have made corresponding changes to the documentation
  - [x] My changes generate no new warnings and I added `TODO:`, `FIXME:`

- [x] REVIEW
  - [x] Add reviewers: The entire team and most influential people in the area/project
  - [x] Assign myself to it

🚨 - `Approve: Wait at least one influential in the project or more experienced`

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ import {
 
 import Row from './components/Row';
 import Separator from './components/Separator';
+import Between from './components/Row Between';
```
Comment (IbraDajani) [inline src/App.tsx:25]:
```suggestion
import RowBetween from './components/RowBetween';
```
We shouldn't have any component with space on the name

File: src/components/Row Between/styles.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components/native';
+
+export const Container = styled.View`
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`;
```
Comment (IbraDajani) [inline src/components/Row Between/styles.ts:7]: we're missing the `width: 100%`

**Review by IbraDajani — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (IbraDajani) [reply src/App.tsx:25]: the component name is `RowBetween`

Comment (IagoWag) [inline src/App.tsx:25]: correct name for the import is `Rowbetween`

**Review by IagoWag — CHANGES_REQUESTED** [review]

**Review by IagoWag — APPROVED** [review]

**Review by IbraDajani — APPROVED** [review]

**Pull request merged**